test/test_sim.py

```diff
@@ -183,7 +183,7 @@ def test_sim_13(datafiles, simulator):
     with datafiles.as_cwd():
         subprocess.check_call(['runSVUnit', '-s', simulator])
 
-        expect_string(br"ERROR: \[50\]\[dut_ut\]: fail_if: svunit_timeout (at `pwd`/./dut_unit_test.sv line:62)", 'run.log')
+        expect_string_esc(br"ERROR: [50][dut_ut]: fail_if: svunit_timeout (at */./dut_unit_test.sv line:62)", 'run.log')
```

Comment:
Do we need to replace `pwd` with `*` in this patch?

Reply:
yes, because in log we have full path to file and this path various depends e.g. if test are run in venv or not, besides in python we don't have `pwd`